core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -83,12 +83,13 @@ object ConsumerGroupCommand extends Logging {
                 printAssignment(assignments, !opts.useOldConsumer)
               case Some("Stable") =>
                 printAssignment(assignments, !opts.useOldConsumer)
+              case None =>
```

Comment:
Thanks for your feedback. I changed the order so the last one in the list (`Some(other)`) is the one we should never reach. With `None` we may still have to do some processing. With that in mind, do you still think `None` should go to the bottom?